### PR TITLE
feat: allow server-manager to bypass log config

### LINKED ISSUE #345: Improve Logs in Version 1.3.0 for Sunodo
## 📚 Context

Sunodo requires the following changes in version 1.3.0:

- Option to not log the config when the node starts.
- Option to the server-manager bypass the log.

### PR DESCRIPTION
**Pull request by torives**
A workaround so Sunodo can display only the application output on its log. Closes #345

### CODE REVIEW

**Review by gligneul — DISMISSED** [review]
Looks good. Let's wait for @endersonmaia approval before merging.

**Review by endersonmaia — APPROVED** [review]
LGTM :rocket:

**Pull request merged**
